diff --git a/Android/android.py b/Android/android.py
index 551168f..a7da68b 100755
--- a/Android/android.py
+++ b/Android/android.py
@@ -30,7 +30,12 @@
 TESTBED_DIR = ANDROID_DIR / "testbed"
 CROSS_BUILD_DIR = PYTHON_DIR / "cross-build"
 
-HOSTS = ["aarch64-linux-android", "x86_64-linux-android"]
+HOSTS = [
+    "aarch64-linux-android",
+    "arm-linux-androideabi",
+    "x86_64-linux-android",
+    "i686-linux-android",
+]
 APP_ID = "org.python.testbed"
 DECODE_ARGS = ("UTF-8", "backslashreplace")
 
